src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
there's no sense on annotating interfaces as classes that are used for instance creation, because we cannot create object from interface